src/Spryker/Zed/Install/Business/InstallFacadeInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Zed\Install\Business;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+interface InstallFacadeInterface
+{
+    /**
+     * Specification:
+     * - Triggers the installation process.
```

Comment:
```suggestion
     * - Triggers an installation process.
```

I think this is more correct as long as install process can differ from env to env